contracts/LendingPool.sol

```diff
@@ -101,7 +101,7 @@ contract LendingPool is OwnableUpgradeable, ERC721Upgradeable, Clone {
     }
 
     function calculateInterest(uint priceOfNextItems) internal view returns (uint interest) {
-        uint borrowed = priceOfNextItems/2 + totalBorrowed;
+        uint borrowed = (priceOfNextItems >> 1) + totalBorrowed;
```

Comment:
good point, does this have any side effects?

Reply:
`DIV` operator costs 5 gas while `SHR` costs 3 gas, so that saves 2 gas. And if that's called multiple times it'd sure amount to much.